handler/user/account.go

```diff
@@ -0,0 +1,79 @@
+package user
+
+import (
+	"encoding/json"
+	"github.com/asynccnu/card_service_v2/handler"
+	"github.com/asynccnu/card_service_v2/pkg/errno"
+	"github.com/asynccnu/card_service_v2/service"
+	"github.com/gin-gonic/gin"
+)
+
+// 输入的表单
+type param struct {
+	Limit string `json:"limit"`
+	Page  string `json:"page"`
+	Start string `json:"start"`
+	End   string `json:"end"`
+}
+
+type tempAccount struct {
+	Result results `json:"result"`
+}
+
+type results struct {
+	Rows []row `json:"rows"`
+}
+
+type row struct {
+	DealName   string  `json:"dealName"`
+	OrgName    string  `json:"orgName"`
+	TransMoney float32 `json:"transMoney"`
+	WalletName string  `json:"walletName"`
+	DealDate   string  `json:"dealDate"`
+	OutMoney   float32 `json:"outMoney"`
+	InMoney    float32 `json:"inMoney"`
+}
+
+// Get gets an account by userid and password
+func Account(c *gin.Context) {
+	// 声明payload变量，因为BindJSON方法需要接收一个指针进行操作
+	var data param
+	var tempLists tempAccount
+	var lists []row
+	var message loginPayload
+
+	if err := c.BindJSON(&message); err != nil {
+		handler.SendError(c, errno.ErrBind, nil, err.Error())
+		return
+	}
+
+	data.Limit = c.Query("limit")
+	data.Page = c.Query("page")
+	data.Start = c.Query("start")
+	data.End = c.Query("end")
```

Comment:
query 有必要去检查下是否传了值，或是给个默认值，比如 limit 为10，可以用 c.DefaultQuery("limit", "10")
另外，既然将 query 放在一个结构体中，那可以直接这样：
```suggestion
	if err := c.BindQuery(&data); err != nil {
	    // ...
	}
```